packages/yoastseo/src/morphology/dutch/generateParticipleForm.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * Adds prefixes and suffixes to the stem.
+ *
+ * @param {string} stemmedWord	The stemmed word for which to create the past participle form.
+ * @param {string} suffix	The suffix to attach at the end of the stem.
+ * @param {string} prefix	The optional prefix.
+ * @param {string} additionalPrefix	An optional additional prefix to attach in front of the first prefix.
+ * @returns {string} The participle form.
+ */
+const addParticipleAffixes = function( stemmedWord, suffix, prefix = "", additionalPrefix = "" ) {
+	return additionalPrefix + prefix + stemmedWord + suffix;
+};
+
+/**
+ *  Checks whether the stem is on the list of stems with a separable prefix that in this specific verb acts
+ *  as inseparable. The exception list includes only those stems that need to take the -d suffix. Those that need
+ *  the -t suffix or no suffix are the same as another form (1st or 3rd person singular) created elsewhere.
+ *
+ * @param {string} stemmedWord The stem to check.
+ * @param {string[]} inseparableExceptions	The list of exception stems.
+ * @returns {string|null} The created participle form or null if the stem was not found on the exception list.
+ */
+const generateInseparableExceptionParticipleForm = function( stemmedWord, inseparableExceptions ) {
+	if ( inseparableExceptions.includes( stemmedWord ) ) {
+		return addParticipleAffixes( stemmedWord, "d" );
```

Comment:
I would simplify it to this:
```suggestion
		return stemmedWord + "d";
```
This communicates directly to the reader of the code that we add the `d` suffix to the end of the stemmed word. Wrapping it into a separate function adds a layer of indirection, in my opinion.